### PR TITLE
Remove password confirmation on registration

### PR DESCRIPTION
**Pull request by egemen-dev**
### Describe your change in plain English.
Removed password confirmation on account creation.

According to the [devise docs](https://github.com/heartcombo/devise/wiki/Disable-password-confirmation-during-registration), this changes are enough to remove password confirmation. 

### Link to the issue
https://github.com/rubyforgood/pet-rescue/issues/91

### CODE REVIEW

**Review by Thrillberg — APPROVED** [review]
Looks great! Thanks for taking this on @egemen-dev! 🎉

**Review by edwinthinks — APPROVED** [review]
